_includes/features.html

```diff
@@ -0,0 +1,100 @@
+                    <ul class="central-list">
+                        <li>Apache Cassandra</li>
+                        <li>Apache HBase</li>
+                        <li>Google Cloud Bigtable</li>
+                        <li>Oracle BerkeleyDB</li>
+                    </ul>
+                </div>
+            </div>
+            <div class="col-sm-4 portfolio-item feature-item">
+                <p class="second-title"><i class="feature-icon fas fa-search"></i> &nbsp; Search
+                </p>
+                <div>
+                    <p class="feature-text">Secondary indexes are supported via:</p>
+                    <ul class="central-list">
+                        <li>Elasticsearch</li>
+                        <li>Apache Solr</li>
+                        <li>Apache Lucene</li>
+                    </ul>
+
+                </div>
+            </div>
+            <div class="col-sm-4 portfolio-item feature-item">
+                <p class="second-title"><i class="fas fa-chart-pie"></i> &nbsp; Analytics
+                </p>
+                <div>
+                    <p class="feature-text">JanusGraph OLAP integrations include:</p>
+                    <ul class="central-list">
+                        <li>Apache Spark</li>
+                        <li>Apache Hadoop</li>
+                        <li>Apache Giraph</li>
+                    </ul>
+                </div>
+            </div>
+            <div class="col-sm-4 portfolio-item feature-item">
+                <p class="second-title"><i class="fas fa-tasks"></i> &nbsp; TinkerPop
+                </p>
+                <div>
+                    <p class="feature-text">Native integration with the Apache TinkerPop™ graph stack:</p>
+                    <ul class="central-list">
+                        <li>Gremlin graph query language</li>
+                        <li>Gremlin graph server</li>
+                        <li>Gremlin applications</li>
+                    </ul>
+                </div>
+            </div>
+            <div class="col-sm-4 portfolio-item feature-item">
+                <p class="second-title"><i class="fas fa-plug"></i> &nbsp; Adapters
+                </p>
+                <div>
+                    <p class="feature-text">There are different 3rd party storage adapters for JanusGraph:</p>
+                    <ul class="central-list">
+                        <li>Couchbase</li>
+                        <li>FoundationDB</li>
+                        <li>DynamoDB</li>
+                    </ul>
+                </div>
+            </div>
+            <div class="col-sm-4 portfolio-item feature-item">
+                <p class="second-title"><i class="fas fa-eye"></i> &nbsp; Visualization
+                </p>
+                <div>
+                    <p class="feature-text">JanusGraph supports variety of visualization tools like Arcade Analytics, Cytoscape, Gephi, Graphexp, Key Lines and Linkurious.</p>
```

Comment:
These used to be links before. Any reason to remove the links here? If they are links again, an unordered list would probably also be a good idea.

Reply:
I can make them to be links again but I am not sure about list. The problem is that if we put all these names into a list it will become bigger then the `<div>` block of feature and the text may overlap with other block in some resolutions. As a fix I can make a text inside Visualization block a little bit smaller then in other blocks. 
The problem:
![Screenshot from 2019-05-14 18-52-01](https://user-images.githubusercontent.com/17673243/57712885-1613e100-767a-11e9-95d4-2955b9b02de9.png)
Possible solution:
![Screenshot from 2019-05-14 18-55-14](https://user-images.githubusercontent.com/17673243/57712903-1f9d4900-767a-11e9-8274-3673a024e4d9.png)

I just thought that it might look a little bit strange if we use different font size in different blocks that is why I've put all names into a line instead of list. What do you think about it?

Reply:
> I just thought that it might look a little bit strange if we use different font size in different blocks that is why I've put all names into a line instead of list.

That's a good point. Having different font sizes would look really weird and it doesn't necessarily have to be a list. We can also keep it as is in this PR.